style.css

```diff
@@ -68,37 +68,61 @@ body {
   display: none;
 }
 
+.toggle {
+  display: block;
+  position: relative;
+  z-index: 9999;
+  cursor: pointer;
+}
+
+.bars {
+  display: block;
+  width: 30px;
+  height: 3px;
+  margin: 5px auto;
+  background-color: #6070ff;
+}
+
+.toggle.open .bars:nth-child(2) {
+  opacity: 0;
+}
+
+.toggle.open .bars:nth-child(1) {
+  transform: translateY(8px) rotate(45deg);
+  background-color: #fff;
+}
+
+.toggle.open .bars:nth-child(3) {
+  transform: translateY(-8px) rotate(-45deg);
+  background-color: #fff;
```

Comment:
- [ ] Please take a look at these images and determine what you need to touch before we go onto the next project, your nav bar items style is a bit not marching the original design, 

1. At the top of the nav page, we have a space.
2. Between the vac bar items, the spacing is not adjusted will according to the original design, please fix it.
3. The font size and the font weight of the nav bar items  is  a bit different from the one in the original design

<img width="756" alt="1" src="https://user-images.githubusercontent.com/77843050/209019899-51c7b939-048f-4d92-9abb-b94aabb7e1ab.png">